src/runtime/google-analytics-event-listener.js

```diff
@@ -65,16 +75,61 @@ export class GoogleAnalyticsEventListener {
     }
 
     // TODO(b/234825847): Remove it once universal analytics is deprecated in 2023.
-    if (typeof this.win_.ga === 'function') {
-      this.win_.ga('send', 'event', gaEvent);
+    const ga = this.win_.ga || null;
+    if (isFunction(ga)) {
+      ga(
+        'send',
+        'event',
+        Object.assign(
```

Comment:
Can we overwrite gaEvent with the analyticsParams and use it across `ga` and `gtag`?
```
// Overwite ...
gaEvent = {
  ... gaEvent,
  ... analyticsParams
}
```

Reply:
The event is slightly different for `ga` and `gtag`, right? `ga` requires `eventCategory` and `eventLabel`, whereas `gtag` requires `event_category`, `event_label`, and the other survey attributes. Is it safe to pass additional attributes to each respective function, i.e. `event_category` to `ga`?